pkg/controller/datavolume/import-controller_test.go

```diff
@@ -85,6 +86,23 @@ var _ = Describe("All DataVolume Tests", func() {
 			}
 		})
 
+		DescribeTable("Should return nil when storage spec has no AccessModes and", func(scName *string) {
+			importDataVolume := newImportDataVolumeWithPvc("test-dv", nil)
+			importDataVolume.Spec.Storage = &cdiv1.StorageSpec{
+				StorageClassName: scName,
+			}
+
+			reconciler = createImportReconciler(importDataVolume)
+			_, err := reconciler.Reconcile(context.TODO(), reconcile.Request{NamespacedName: types.NamespacedName{Name: "test-dv", Namespace: metav1.NamespaceDefault}})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Maybe also
`Expect(result).To(Equal(reconcile.Result{}))`
To ensure requeue/requeueafter don't get set?

Reply:
done